libs/partners/google-vertexai/langchain_google_vertexai/llms.py

```diff
@@ -227,8 +253,12 @@ def validate_environment(cls, values: Dict) -> Dict:
             )
         else:
             if is_gemini:
-                values["client"] = model_cls(model_name=model_name)
-                values["client_preview"] = preview_model_cls(model_name=model_name)
+                values["client"] = model_cls(
```

Comment:
same as for chat_models, allow passing `safety_filters` through kwargs to `_generate`

Reply:
done